pkgs/shells/fish/plugins/sdkman-for-fish.nix

```
@@ -0,0 +1,20 @@
+{ lib, buildFishPlugin, fetchFromGitHub }:
+
+buildFishPlugin rec {
+  pname = "sdkman";
```

```suggestion
  pname = "sdkman-for-fish";
```
The `pname` should be the same as the upstream.
https://nixos.org/manual/nixpkgs/unstable/#sec-package-naming

---

nice catch - I missed that